### ISSUE TITLE
Add documentation about MIR and mir_bridge functionalities

### ISSUE THREAD

**Issue by matthias-mayr** [DFKI-NI/mir_robot#59]
We recently got a new MIR200 and since it comes with a new software version, it was not directly compatible with this ROS driver.

We spent quite some time to figure out, how the MIR would work, how it connects with rosbridge and why it was not working right away. With the insight we obtained up to now, we could have saved quite some of it, if there would have been some more documentation, e.g.

- That MIR is actually running ROS internally (although that didn't really take long to figure out)
- A figure of the architecture like `ROS - mir_driver - rosbridge - rosbridge - MIR`
- That one [can actually just connect to the roscore on MIR](https://github.com/dfki-ric/mir_robot/issues/45#issuecomment-655979633). I misread the network specifications and thought that all the ports would be closed, so I though our only access would be through rosbridge.

All the manuals I got with the MIR leave out, that it runs ROS and a lot of other tips and tricks that appear in various issues here. It seems to me that it would be a lot of help for people if this implicit knowledge would be expressed somewhere.
What do you think?

**Comment by mintar** [comment]
Yes, any updates to the documentation would be more than welcome!

**Comment by matthias-mayr** [comment]
Do you prefer a set of markdown files or to start a wiki?

**Comment by mintar** [comment]
I have a slight preference for a set of markdown files.

**Comment by liesrock** [comment]
Hi @matthias-mayr, @mintar I can help on this as well.

@matthias-mayr can you provide the above documentation even if is just a draft?

**Comment by matthias-mayr** [comment]
This issue is still in the back of my head, but I am totally busy until the first week of November.

**Comment by matthias-mayr** [comment]
![mir_architecture](https://user-images.githubusercontent.com/64466074/106586889-3fa92600-6549-11eb-8518-6160f09f68e0.jpg)

For convenience a jpg export here. Comments are appreciated.
